docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -141,6 +143,20 @@ configuration). If the subsystem detects that the `OIDC` mechanism is indeed req
 activate the authentication mechanism automatically. Otherwise, no activation will occur and deployment
 will continue normally.
 
+=== Scope Configuration
+
+OpenID Connect allows the client to use scope values to request access to specific privileges. The scopes determine if the client will have access to additional resources when sending a request. According to the https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims[documentations], there are a few scope values that OpenID connect allows, such as: 
```

Comment:
s/privileges/privileges for access tokens.

Reply:
s/The scopes determine if the client will have access to additional resources when sending a request/An access token's scopes determine what resources will be available when using the access token to access an OAuth2 protected endpoint.

Reply:
"According to the https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims[documentations], there are a few scope values that OpenID connect allows, such as:"

Maybe could reword to something like:

The OpenID Connect https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims[specification] defines a few scope values including:

Reply:
fixed!